proto/valorem/trade/v1/sq.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+import "types.proto";
+
+package valorem.trade.v1;
+
+/**
+ * Soft Quotes (SQ) services and related message formats.
+ */
+
+message SoftQuoteRequest {
+    /** The exercise token address */
+    H160 exercise = 1;
+
+    /** The underlying token address */
+    H160 underlying_asset = 2;

```

Comment:
This would be better keyed in sum on either the token_id from the erc1155 or the claim.

Here is how we think about an option position in the new FE data model

```
type OptionKeyPayload = {
        underlyingAsset: Address,
        underlyingAmount: bigint,
        exerciseAsset: Address,
        exerciseAmount: bigint,
        exerciseTimestamp: bigint,
        expiryTimestamp: bigint
}

type Option: {
    optionKeyPayload?: OptionKeyPayload;
    claimKey: bigint = 0n;
  isClaim = false;
  isCreated?: boolean
};

enum Action { BUY, SELL, CREATE, WRITE, EXERCISE, REDEEM }
```

Reply:
So my assumption here was that the option will not exist since the taker will be working out a strike price they want to have and they want to approximately see what the price for these options would be. 

I wasn't able to work out how get the token identifiers without having that token id existing within clearinghouse and with the uri generator it seemed like I would need to pass in pretty much everything for the option anyways.

Is there are better way to get the option details?